lib/Model.php

```diff
@@ -415,7 +415,7 @@ public function __set($name, $value)
 		foreach (static::$delegate as &$item)
 		{
 			if (($delegated_name = $this->is_delegated($name,$item)))
-				return $this->$item['to']->$delegated_name = $value;
+				return $this->{$item['to']}->$delegated_name = $value;
```

Comment:
Yeah may as well. Did a search proj-wide and there's a tonnnnn of these. Butmay as well driveby fix here 👍